sections/03_categorize.md

```diff
@@ -44,92 +44,76 @@ regards to data availability and labeling.
 
 #### Imaging applications in healthcare
 
-One area where deep learning methods have had substantial success has been in
-image analysis. Applications in areas of medicine that use imaging extensively
-are also emerging. To the date, deep learning has been employed for a wide range
-of tasks in medical imaging, including classification of exams and
-lesions/nodules, localization of organs, regions, landmarks, and lesions,
-segmentation of organs, organ substructures, and lesions, medical image
-registration, content-based image retrieval, image generation and enhancement,
-and combining image data with clinical reports [@tag:Litjens2017_medimage_survey
+Deep learning methods have transformed the analysis of natural photographs and video,
+and similar examples are beginning to emerge with medical images.
+Deep learning has been used to classify
+lesions and nodules; localize organs, regions, landmarks and lesions;
+segment organs, organ substructures and lesions;
+retrieve images based on content; generate and enhance images;
+and combine images with clinical reports [@tag:Litjens2017_medimage_survey
 @tag:Shen2017_medimg_review].
 
-Closest to natural images are applications of deep learning aimed at detection
-and recognition of melanoma, the deadliest form of skin cancer. Recent works
-included applications to both dermoscopy [@tag:Codella2016_ensemble_melanoma
-@tag:Yu2016_melanoma_resnet] and non-dermoscopic clinical photography images of
-skin lesions [@tag:Jafari2016_skin_lesions @tag:Esfahani2016_melanoma
-@tag:Esteva2017_skin_cancer_nature]. For both modalities pre-training on natural
-images appears to be common model initialization that allows the use of very
-deep networks without overfitting. Reported performance is competitive or better
-compared to a board of certified dermatologists
-[@tag:Codella2016_ensemble_melanoma @tag:Esteva2017_skin_cancer_nature]. This
-approach is known as transfer learning (see Discussion).
-
-Another fast emerging area of deep learning method applications is the detection
-of ophthalmological diseases such as diabetic retinopathy and age-related
-macular degeneration. Diagnosis of diabetic retinopathy through color fundus
-images became of interest for deep learning researchers and practitioners after
-the large labeled image set was made publicly available during the corresponding
-2015 Kaggle competition [@tag:Pratt2016_dr]. Most attempts included training a
-network from scratch [@tag:Pratt2016_dr @tag:Abramoff2016_dr
-@tag:Leibig2016_dr], while Gulshan et al. [@tag:Gulshan2016_dt] employed
+Though there are many commonalities with the analysis of natural photographs,
+there are also key differences. In all cases that we examined, fewer than one
+million images were available for training, and datasets are often many orders
+of magnitude smaller than collections of natural photographs. To address this
+challenge, researchers addressing different subtasks have employed different
```

Comment:
Let's replace the repeated "different".  Maybe "Researchers have developed subtask-specific strategies to address this challenge."